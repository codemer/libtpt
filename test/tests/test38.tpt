@set(val.one, "apple")\
@set(val.two, "orange")\
@set(val.three, "pear")\
@set(val.four, "tangerine")\
@keys(keys, val)\
@size(keys)
@foreach key (keys) {
${key} = ${val.${key}}
}
