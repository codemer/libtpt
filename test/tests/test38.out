4
four = tangerine
one = apple
three = pear
two = orange
